pmtiles/server.go

```diff
@@ -294,6 +296,7 @@ func (server *Server) getTileJSON(ctx context.Context, httpHeaders map[string]st
 	}
 
 	httpHeaders["Content-Type"] = "application/json"
+	httpHeaders["Etag"] = generateEtag(tilejsonBytes)
```

Comment:
I think the end result is case-insensitive but the capitalization convention seems to be `ETag` with a capital T, any objections to changing it to that?